server/service_windows.go

```diff
@@ -66,8 +67,16 @@ func (w *winServiceWrapper) Execute(args []string, changes <-chan svc.ChangeRequ
 	status <- svc.Status{State: svc.StartPending}
 	go w.server.Start()
 
+	if v, exists := os.LookupEnv("NATS_STARTUP_DELAY"); exists {
+		delay, err := time.ParseDuration(v)
```

Comment:
`if delay, err := time.ParseDuration(v); err == nil {`

Reply:
yes, I don't see why I did not do it.